content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_storages.rb

```diff
@@ -0,0 +1,46 @@
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Transform
+          module Import
+            class ListStorages
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                values_hash = {}
+                values_hash[''] = '-- select storage from list --'
```

Comment:
@matobet 
I think the key here should be nil
``
value_hash[nil] = '-- select storage from list --'
``
On line 21 you are setting nil to None if no providers are found.